package-testing/php-sdk-relay/.env.EXAMPLE

```diff
@@ -1,6 +1,5 @@
 # Set your appropriate values and save as `.env`
-EPPO_API_HOST=localhost
-EPPO_API_PORT=5000
+EPPO_BASE_URL=localhost:5000
```

Comment:
Here's the default base url we have in SDKs: `https://fscdn.eppo.cloud/api`. Would be great to have the same format here (add protocol + trailing `/api`)